### DISCUSSION TITLE
Can Laravel Cashier be integrated into the admin panel?

### DISCUSSION THREAD

**Discussion by carrilhorafael57** [filamentphp/filament#8465] [Help]
Greetings,

I'm currently in the midst of building my admin panel using Filament. I'm exploring the possibility of integrating Laravel Cashier into my project and would be grateful for insights on the feasibility of incorporating transaction management and related features into the Filament admin panel.

I'm relatively new to both Laravel Filament and Laravel itself, so any assistance is greatly appreciated.

Thank you for your guidance!

**Comment by leandrocfe** [comment]
I've never tried this before but it is possible I think.

**Comment by jamie-syon** [comment]
@carrilhorafael57 Did you have any luck?  I'm trying but getting no events firing on the paddle_button links
